testutils/utils.go

```diff
@@ -250,17 +242,30 @@ func CreateContainerNamespaceWithCid(container_id string) (containerNs ns.NetNS,
 }
 
 func CreateContainer(netconf, podName, podNamespace string, ip string) (container_id string, session *gexec.Session, contVeth netlink.Link, contAddr []netlink.Addr, contRoutes []netlink.Route, targetNs ns.NetNS, err error) {
-	return CreateContainerWithId(netconf, podName, podNamespace, ip, "")
+	targetNs, container_id, err = CreateContainerNamespace()
+
+	if err != nil {
+		return "", nil, nil, nil, nil, nil, err
+	}
+
+	session, contVeth, contAddr, contRoutes, err = RunCNIPluginWithId(netconf, podName, podNamespace, ip, container_id, "", targetNs)
+
+	return
 }
 
 // Create container with the giving containerId when containerId is not empty
+//
+// Deprecated: Please call CreateContainerNamespace and then RunCNIPluginWithID directly.
 func CreateContainerWithId(netconf, podName, podNamespace, ip, containerId string) (container_id string, session *gexec.Session, contVeth netlink.Link, contAddr []netlink.Addr, contRoutes []netlink.Route, targetNs ns.NetNS, err error) {
-	targetNs, container_id, err = CreateContainerNamespaceWithCid(containerId)
-
+	targetNs, container_id, err = CreateContainerNamespace()
```

Comment:
`container_id` is python style (underscores) - I know it was already this way, but....

I'd suggest calling this something like `containerID` and then the arg that's passed in can be `givenContainerID` or something like that to differentiate them.

Reply:
(Similar for `targetNs` which should be `targetNS`)